web/modules/custom/ai_screening_project/src/Helper/ProjectHelper.php

```diff
@@ -400,6 +405,22 @@ public function loadProject(string $id): ?NodeInterface {
     return $this->isProject($node) ? $node : NULL;
   }
 
+  /**
+   * Preprocess node project event.
+   *
+   * @param \Drupal\preprocess_event_dispatcher\Event\NodePreprocessEvent $event
+   *   The event being performed.
+   */
+  public function preprocessProject(NodePreprocessEvent $event): void {
+    $variables = $event->getVariables();
+    $node = $variables->getEntity();
+    if ($node instanceof NodeInterface) {
+      $variables->set('projectTracks', $this->loadProjectTracks($node));
+      $variables->set('projectGroup', $this->loadProjectGroup($node));
+      $variables->set('projectMembers', $this->loadProjectGroup($node)->getRelatedEntities('group_membership'));
```

Comment:
It may be easier to add these helper functions (`load…`) in `ProjectHelper` and inject the helper into the template and use the functions in the template. Or use https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_entity_load/11.x to set (virtual) fields on the node to make the related entities readily available.

But we can look at that later …